x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The hottest login theme around for KDE Plasma 5"
+HOMEPAGE="https://github.com/MarianArlt/kde-plasma-chili"
+SRC_URI="https://github.com/MarianArlt/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	kde-plasma/plasma-meta
+	x11-misc/sddm
+"
+
+DOCS=(
+	AUTHORS
+	CHANGELOG.md
+	CREDITS
+	README.md
+)
+
+src_install() {
+	einstalldocs
+
+	rm "${S}/${DOCS[@]}" "${S}/LICENSE.md"
```

Comment:
I don't think this is necessary, if you don't use globbing to install the theme below. If you can't work around it, append with `|| die` and preferably do this operation in `src_prepare` phase.

Reply:
It has to be run after _einstalldocs_: and therefore in _src_install_.